x-pack/solutions/observability/plugins/profiling/public/services.ts

```diff
@@ -132,7 +127,7 @@ export function getServices(): Services {
       return hasSetup;
     },
     postSetupResources: async ({ http }) => {
-      await http.post(paths.HasSetupESResources, {});
+      await http.post(paths.HasSetupESResources, { body: JSON.stringify({}) });
```

Comment:
This to send an empty body to the POST.
Not sure why but if we don't the request fails as it finds a `null` body

Reply:
yeah, apparently...